config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml

```
@@ -79,4 +79,5 @@ spec:
   maturity: beta
   provider:
     name: Red Hat
+  replaces: rhpam-kogito-operator.v7.11.0
   version: 7.11.0
```

```suggestion
  version: 7.11.0-1
```
Should this also be updated?

---

`config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml` file is just used as a template to create actual bundle file. Version field (`version: 7.11.0`) of this file will be overriden when `make bundle` cmd run. Value of this field is controlled through make file.

---

OK, no problem.